scripts/debug-cli/src/main.rs

```diff
@@ -169,3 +180,45 @@ fn decode_hex_print<T: Decode + Debug>(hex_data: &str) -> T {
     println!("Decode: {:?}", message);
     message.unwrap()
 }
+
+type AccountId = sp_runtime::AccountId32;
+type Balance = u128;
+
+fn decode_mq_payload(destination: &str, payload: &[u8]) {
```

Comment:
Change `&str` to `&[u8]` and move it to some other crate so that pRuntime can reuse it?

Reply:
Couldn't find a good place.

`phala-mq` is not aware of the types defined in `common/types` (the latter one depends on mq). `common/types` is already mixed with too much stuff, and it still stay generic about the chain types (e.g. Balance, AccountId). So I guess it's not a good place as well.

Do you think we should create another crate under `crates/`, with all the generic types populated for a typical Phala blockchain, and only use when necessary?

Reply:
> Do you think we should create another crate under `crates/`, with all the generic types populated for a typical Phala blockchain, and only use when necessary?

Ah, sounds like a big refactor. So I think we may just leave a TODO here in this PR.

Reply:
Good idea